src/librustdoc/html/static/main.js

```diff
@@ -16,6 +16,14 @@
 (function() {
     "use strict";
 
+    Object.nbElements = function(obj) {
```

Comment:
Why do you define this on `Object` (of all things) and not just as a free function?

Reply:
I preferred it as is but I can make a free function if you prefer.

Reply:
Yeah, I don't think adding a static method for this on a core, well, `Object` is the way to go. This function probably shouldn't escape the scope of this file anyway.